peer/brontide.go

```diff
@@ -677,7 +670,65 @@ func (p *Brontide) Start() error {
 
 		return lastSerializedBlockHeader[:]
 	}
-	go p.pingHandler(newPingPayload)
+	// TODO(roasbeef): make dynamic in order to
+	// create fake cover traffic
+	// NOTE(proofofkeags): this was changed to be
+	// dynamic to allow better pong identification,
+	// however, more thought is needed to make this
+	// actually usable as a traffic decoy
+	randPongSize := func() uint16 {
+		return uint16(
+			// We don't need cryptographic randomness here.
+			/* #nosec */
+			rand.Intn(lnwire.MaxPongBytes + 1),
+		)
+	}
+
+	p.pingManager = NewPingManager(&PingManagerConfig{
+		NewPingPayload:   newPingPayload,
+		NewPongSize:      randPongSize,
+		IntervalDuration: pingInterval,
+		TimeoutDuration:  pingTimeout,
+		SendPing: func(ping *lnwire.Ping) {
+			msgWritten := make(chan error)
+			p.queueMsg(ping, msgWritten)
+			err, ok := <-msgWritten
+			if err != nil {
+				eStr := "writeHandler failed to write " +
+					"ping to the wire: %v"
+				p.log.Errorf(eStr, err)
+				p.Disconnect(fmt.Errorf(eStr, err))
+
+				return
+			}
+			if !ok {
+				eStr := "writeHandler prematurely" +
+					" closed confirmation channel"
+				p.log.Error(eStr)
+				p.Disconnect(fmt.Errorf(eStr))
+
+				return
```

Comment:
is it important that we initiate a disconnect if we cant send a ping? I think we can just make this:

```
SendPing: func(ping *lnwire.Ping) {
			p.queueMsg(ping, nil)
}
```